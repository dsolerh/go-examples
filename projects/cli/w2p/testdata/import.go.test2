package example

// replace: do/not/exist -> exist
import "github.com/package/do/not/exist/three"

func example() {
    three.Print()
}