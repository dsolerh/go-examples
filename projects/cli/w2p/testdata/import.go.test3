package example

// replace: do/not/exist -> exist
import _ "github.com/package/do/not/exist/three"

func example() {
}