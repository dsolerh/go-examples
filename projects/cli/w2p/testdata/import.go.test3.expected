package example

// replace: do/not/exist -> exist
import _ "github.com/package/exist/three"

func example() {
}